Added FRC team 2412's github repository

robototes2412 opened the pull request:
Added link to team 2412's github repository (https://github.com/robototes) to team page

robototes2412 committed:
Added FRC team 2412's github repository

Added link to team 2412's github repository (https://github.com/robototes) to team page

virtuald commented:
This isn't quite correct. Your github site should be under links, and then the link to the actual code repo should be in the 2016 section. Does that make sense?

robototes2412 committed:
Added FRC team 2412's github repository - fixed

Put github site (https://github.com/robototes) under links section, put repository in 2016 section.

robototes2412 commented:
Yeah it does make sense. I guess I didn't notice that earlier. Thanks for pointing it out!

virtuald commented:
Yeah, I still need to document this stuff.  Thanks!
